cypress/e2e/base.spec.ts

```diff
@@ -20,4 +20,31 @@ describe('Visit directory page and verify users exists', () => {
   });
 });
 
+describe('Onboarding Page', () => {
+  afterEach(() => {
+    cy.truncateDatabase();
+  });
+
+  it('displays the correct heading', () => {
```

Comment:
This test will fail once we make `/onboarding` only accessible to logged-in users. Might want to add a `cy.login();` before the `visit()`.

That said, this isn't really testing much... do we even need it?

Reply:
I mean, idk what the threshold for testing really is